MemoirVerse.UI/login.php

```diff
@@ -1,28 +1,28 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>MemoirVerse Login</title>
-    <link rel="stylesheet" href="./index.css"/>
-  </head>
-  <body>
+    <link rel="stylesheet" href="login.css">
```

Comment:
@RosarioAngelica 
can you please add "./login.css" so that it will be functional to all types of browser.